ui/src/components/stepforms/widgets/Autocomplete.vue

```diff
@@ -29,7 +30,14 @@
           <span class="option__title">{{ props.option.label }}</span>
         </template>
         <template v-if="options[0] && options[0].label" slot="option" slot-scope="props">
-          <div class="option__container" :title="props.option.tooltip">
+          <div
+            class="option__container"
+            :class="{
+              'option__container--disabled': props.option.disabled,
+            }"
+            :title="props.option.tooltip"
```

Comment:
Oh OK! I though you were gonna use v-tooltip, as in the rest of the app. But in tat case you should have removed the `pointer-events: none` I guess.